src/worker/tasks/mineTransactionWorker.ts

```diff
@@ -128,6 +128,19 @@ const _mineTransaction = async (
     if (result.status === "fulfilled") {
       const receipt = result.value;
       job.log(`Found receipt on block ${receipt.blockNumber}.`);
+
+      // removed nonce from sentnonce set
+      const removed = await redis.srem(
+        `sentnonce:${receipt.from.toLowerCase()}:${chainId}`,
```

Comment:
we should create a helper like in the case of `recycledNoncesKey()` to fetch the key everywhere

Reply:
Can we use this pattern `nonce-sent:1993:0x...`